pom.xml

```diff
@@ -130,8 +130,8 @@
 			</dependency>
 			<dependency>
 				<groupId>org.apache.velocity</groupId>
-				<artifactId>velocity</artifactId>
-				<version>1.6.2</version>
+				<artifactId>velocity-engine-core</artifactId>
```

Comment:
wondering why you are changing to velocity-engine-core

Reply:
@HerbertYiga the artifactId was changed to velocity-engine-core 2.1 
https://mvnrepository.com/artifact/org.apache.velocity/velocity-engine-core/2.1